src/test/ut/ut_cortxfs_xattr_file_ops.c

```diff
@@ -77,6 +85,7 @@ int main(void)
 	int rc = 0;
 	char *test_log = "/var/log/cortx/test/ut/xattr_file_ops.log";
 
+	perfc_tsdb_setup();
```

Comment:
I thought we will not do addb profiling from UT?

Reply:
Yes, I enabled only for this specific test to show how to enable addb tracing for UTs. It can be used as an example in future. But if you think this needs to be disabled, I can comment out line number 88.

Reply:
Let me comment out this in the UT.

Reply:
ok